src/components/cc-orga-member-list/cc-orga-member-list.stories.js

```diff
@@ -45,6 +45,20 @@ const authorisationsAdmin = {
   edit: true,
   delete: true,
 };
+const baseItem = {
+  authorisations: authorisationsAdmin,
+  members: {
+    state: 'loaded',
+    value: baseMemberList,
+    identityFilter: '',
+    mfaDisabledOnlyFilter: false,
+    dangerZoneState: 'idle',
+  },
+};
+
+/**
+ * @typedef {import('./cc-orga-member-list.js').CcOrgaMemberList} CcOrgaMemberList
+ */
```

Comment:
fix: need to move these after `export default` because it breaks storybook docs  (I don't have a better solution since it's related to Storybook's code)